src/Microsoft.IdentityModel.Tokens/X509CertificateHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Helper class to load X509Certificate2 from byte array.
+    /// </summary>
+    internal class X509CertificateHelper
+    {
+        public static X509Certificate2 Load(byte[] data)
+        {
+#if NET9_0_OR_GREATER
```

Comment:
This API is available in net8 are we choosing net9 deliberately?

Reply:
The breaking change was introduced in net9

Reply:
Talked offline and looks like we prefer to use it in net8 - updating.

Reply:
Update: ![image](https://github.com/user-attachments/assets/9db3bc9e-bdb2-48e9-8113-964216ef83ea)

Reply:
@pmaytak found that: 

For .NET 9, in the Definition, the assembly is SystemSecurity.Cryptography.dll.
[X509CertificateLoader.LoadCertificate Method (System.Security.Cryptography.X509Certificates) | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificateloader.loadcertificate?view=net-9.0#definition)
 
For .NET 8, it's the backwards compat package: Microsoft.Bcl.Cryptography.dll
[X509CertificateLoader.LoadCertificate Method (System.Security.Cryptography.X509Certificates) | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificateloader.loadcertificate?view=net-8.0#definition)

I would rather not introduce a new dependency to also use this api in .net 8. Thoughts?

Reply:
Discussed offline and we don't want to add another dependency to use this in .net 8.